index.html

```diff
@@ -11,54 +11,153 @@
 </head>
 
 <body>
-    <header>
-        <nav>
-            <div class="logo">
-                <a href="#">Jonah.</a>
-            </div>
-            <div class="mobile-menu">
-                <a href="#"><i class="fas fa-bars"></i></a>
-            </div>
-        </nav>
-    </header>
-    
     <div class="container">
         <main>
-            <section class="headline">
-                I'm Jonah<br>Glad to see you!
-            </section>
-
-            <section class="body">
-                <p>
-                    I’m a software developer! I can help you build a product feature or website. Look through some
-                    of my work and experience! If you like what you see and have a project you need coded, don’t
-                    hesitate to contact me.
-                </p>
-            </section>
-
-            <section class="social-media">
-                <p>LET'S CONNECT</p>
```

Comment:
- [ ] I noticed that you deleted the header and the first section you did before, you should keep them.